src/components/CodeEditor/index.jsx

```diff
@@ -3,6 +3,7 @@ import { string, func, bool } from 'prop-types';
 import { Controlled as CodeMirror } from 'react-codemirror2';
 import 'codemirror/lib/codemirror.css';
 import 'codemirror/theme/material.css';
+import 'codemirror/mode/javascript/javascript';
```

Comment:
Is this what's doing the virtualization? I can't find much information on how codemirror works (eg: https://codemirror.net/mode/javascript/index.html is pretty sparse).

Reply:
Virtualization works based on the height of the editor. Showing all of the editor was not virtualizing anything since the full DOM was displayed.